.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 /artifacts
 /cache
 /contracts-exposed
+.DS_Store
```

Comment:
You should add this in a system-wide gitignore file rather than in each project.

Reply:
oops. will do yea tx